aggregator/src/datastore.rs

```diff
@@ -2149,6 +2149,92 @@ WITH updated as (
         WHERE
             -- Constraint for tasks table in FROM position
             tasks.id = collect_jobs.task_id
+            -- Only return
```

Comment:
Are there some words missing from this sentence?
```suggestion
            -- Only return time interval collect jobs.
```